server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckBaseUtility.java

```diff
@@ -170,6 +165,42 @@ private void throwException(
     }
   }
 
+  private void checkDuplicateEntries(
+      ParserRuleContext ctx,
+      Map<Integer, TerminalNode> entries,
+      Map<String, ErrorSeverity> specialSeverities) {
+    if (ctx.getChildCount() != 0) {
+      for (ParseTree entry : ctx.children) {
+        if (entry.getChildCount() == 0) {
+          TerminalNode current = (TerminalNode) entry;
+          if (current.getSymbol().getType() < 897) {
```

Comment:
I assume that 897 is token id? This, in general, does not look like a good idea, the tokens could be reordered for number of reasons and this condition will change its meaning.

I think you'd need to construct a table of tokens you want to check using `CICSLexer.tokenname`.